server/models/user.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const validate = require('mongoose-validator');
+
+const Schema = mongoose.Schema;
+
+// Email validator
+const emailValidator = validate({
+  validator: 'isEmail',
+  message: 'Provide a proper email address.'
+});
+
+// Create User schema
+const UserSchema = new Schema(
+  {
+    email: {
+      type: String,
+      required: true,
+      validate: emailValidator
+    },
+    password: { type: String, required: true }
```

Comment:
This is a personal preference, but I actually think when you change structures that are bumped up together like this - it makes it slightly more difficulty to read the code as you're scrolling down. I'm a fan of consistency, but I'm open to feedback on that!

Reply:
I totally agree, and have updated.  I thought it was a linter issue, and was waiting til we have the project lint file, but I was able to change.